File: repos_estudiantes_ALVAREZ_primera_entrega/GARNICA_FUGUEROA_CAMILA_NATALIA/Introduccion a POO/Ejercicio 1/1.Codigos/src/Ejercicio_1/Persona.java
package Ejercicio_1;

public class Persona {
	public String nombre;
	public int edad;
	public String ciudad;
	
	public Persona(String nombre, int edad, String ciudad) {
		this.nombre = nombre;
		this.edad = edad;
		this.ciudad = ciudad;
	}

	public void saludar() {
		System.out.println("Hola soy " + nombre + " de " + ciudad);
	}
	
	public void mayor () {
		if (edad >=18) {
			System.out.println(nombre + " con " + edad + " es mayor de edad");
		}
		else {
			System.out.println(nombre + " con " + edad + " es menor de edad");
		}
	}
}

File: repos_estudiantes_ALVAREZ_primera_entrega/GARNICA_FUGUEROA_CAMILA_NATALIA/Introduccion a POO/Ejercicio 1/1.Codigos/Persona.py
class Persona:
    def __init__(self, nombre, edad, ciudad):
        self.nombre = nombre
        self.edad = edad
        self.ciudad = ciudad 

    def saludo(self):
        return f"Hola, soy {self.nombre} de {self.ciudad}" 

    def verificacionEdad(self):
        if self.edad >= 18:
            return "Es mayor de edad"
        else:
            return "Es menor de edad"

persona1 = Persona("Amy", 16, "Argentina")
print(persona1.saludo())
print(persona1.verificacionEdad())  

print()

persona2 = Persona("America", 18, "Rusia")
print(persona2.saludo())
print(persona2.verificacionEdad())

print()

persona3 = Persona("Mimica", 30, "Brasil")
print(persona3.saludo())
print(persona3.verificacionEdad())